plugins/catalog-react/src/index.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export * from '@backstage/catalog-client';
+export { catalogApiRef } from './api';
+export * from './components';
+export * from './hooks';
+export {
+  catalogRouteRef,
+  entityRoute,
+  entityRouteParams,
+  entityRouteRef,
+  rootRoute,
+} from './routes';
```

Comment:
Hmm. I was going to write a comment about `externalRoute` and moving the refs back to the plugin package ... but I see these are actually in active use. So until we clean things up I think these may actually be correct to move out.

Reply:
I mean the ones in `-react` could actually have been renamed `legacy*` and marked deprecated or something. But maybe it's out of scope for the PR...

Reply:
Yeah, we have to update all cases where these routes are in use, I think it's better to do it in a separate PR